test/metabase/http_client.clj

```diff
@@ -238,8 +243,13 @@
     (cond-> parsed
       ;; un-nest {:request-options {:request-options <my-options>}} => {:request-options <my-options>}
       (:request-options parsed) (update :request-options :request-options)
-      ;; convert query parameters into a flat map [{:k :a, :v 1} {:k :b, :v 2}] => {:a 1, :b 2}
-      (:query-parameters parsed) (update :query-parameters (partial into {} (map (juxt :k :v)))))))
+      ;; convert query parameters into a flat map [{:k :a, :v 1} {:k :b, :v 2} {:k :b, :v 3}] => {:a 1, :b [2 3]}
+      (:query-parameters parsed) (update :query-parameters (fn [query-params]
```

Comment:
this allows we to do

```
(mt/http-user-http ... :param1 "a" :param1 "b")
```

which is equivalent of `?param1=a&param1=b` in the URL.